requirements.txt

```diff
@@ -3,4 +3,5 @@ Flask-Cors==3.0.7
 pandas==0.25.3
 requests==2.22.0
 xlrd==1.2.0
-psycopg2
\ No newline at end of file
+psycopg2
+ibm-watson>=5.3.0
```

Comment:
It would be nice to also include https://pypi.org/project/ibm-cloud-sdk-core/ -- i'm guessing the watson library will pull it in but it's nice to be explicit since it's directly imported in the code

```python
from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
```

Reply:
Added :)